tools/mmp/Assembly.cs

```diff
@@ -21,5 +25,52 @@ bool CopyAssembly (string source, string target)
 
 			return copied;
 		}
+
+		public void ComputeSatellites ()
+		{
+			var path = Path.GetDirectoryName (FullPath);
+			var satellite_name = Path.GetFileNameWithoutExtension (FullPath) + ".resources.dll";
+
+			foreach (var subdir in Directory.GetDirectories (path)) {
+				var culture_name = Path.GetFileName (subdir);
+				CultureInfo ci;
+
+				if (culture_name.IndexOf ('.') >= 0)
+					continue; // cultures can't have dots. This way we don't check every *.app directory
+
+				try {
+					ci = CultureInfo.GetCultureInfo (culture_name);
+				} catch {
+					// nope, not a resource language
+					continue;
+				}
+
+				if (ci == null)
+					continue;
+
+				var satellite = Path.Combine (subdir, satellite_name);
+				if (File.Exists (satellite)) {
+					if (Satellites == null)
+						Satellites = new List<string> ();
+					Satellites.Add (satellite);
+				}
+			}
+		}
+
+		public void CopySatellitesToDirectory (string directory)
+		{
+			if (Satellites == null)
+				return;
+
+			foreach (var a in Satellites) {
+				string target_dir = Path.Combine (directory, Path.GetFileName (Path.GetDirectoryName (a)));
+				string target_s = Path.Combine (target_dir, Path.GetFileName (a));
+
+				if (!Directory.Exists (target_dir))
+					Directory.CreateDirectory (target_dir);
+
+				CopyAssembly (a, target_s);
+			}
```

Comment:
That's a copy of the same method in `tools/common/Assembly.cs`. The code should be moved into `tools/common/Assembly.cs` which is where the shared mtouch/mmp code aside.